src/client/broadcast.ts

```diff
@@ -40,23 +40,12 @@ class BroadcastClient extends Client {
     })
 
     clients.forEach((client) => {
-      client.on('ledger', this.onLedgerEvent.bind(this))
       client.on('error', (errorCode, errorMessage, data) =>
         this.emit('error', errorCode, errorMessage, data)
       )
     })
   }
 
-  onLedgerEvent(ledger) {
```

Comment:
Why are we getting rid of this function?

Reply:
This method is only being used because the client was initially automatically subscribed to ledger events, but that's not really necessary. Now that it's not, this method doesn't do anything, so it should be removed.

Reply:
Gotcha, so using `server_info` to fetch the range of ledgers, instead of listening to the subscriptions.

Reply:
Yep, or just making a `ledger` call to get information about a particular ledger